### PR TITLE
Update Listener to bind (Closes #1)

### LINKED ISSUE #1: Improve document title
The first time I heard the title the thought that crossed my mind was that it could be used to support more modes of UDP usage that focus on listening - such as multicast reception.

However, when I look at the draft, the interaction model doesn't really fit the multicast subscribe/join model. (I can think of ways to do that but I won't waste your time here).

Assuming you want to keep the document scope and design as is, I might suggest a tweak to the document title that more closely matches that definition. Maybe something like "UDP socket multiplexing over single HTTP requests" :bike::shed"

### PR DESCRIPTION
**Pull request by asingh-g**
(Closes #1)

We will likely also need to update the repo name and consequently the url inside the draft once this change is submitted

### CODE REVIEW

File: draft-ietf-masque-connect-udp-listen.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Proxying Listener UDP in HTTP"
-abbrev: "CONNECT-UDP Listen"
+title: "Proxying Binding UDP in HTTP"
```
Comment (DavidSchinazi) [inline draft-ietf-masque-connect-udp-listen.md:2]: Should we say "Binding" or "Bound"?

**Review by DavidSchinazi — COMMENTED** [review]
Similarly, can we add "Closes #1" to the PR description?

File: draft-ietf-masque-connect-udp-listen.md (hunk 1 above)
Comment (asingh-g) [reply draft-ietf-masque-connect-udp-listen.md:2]: Either way makes sense, as binding is an adjective here. I don't lean strongly either way.

Comment (DavidSchinazi) [reply draft-ietf-masque-connect-udp-listen.md:2]: Cool. I'd say I have a preference for Bound if they're the same to you

**Review by DavidSchinazi — APPROVED** [review]

**Pull request merged**